borderlands3/bl3_vip_code_redeem.js

```diff
@@ -63,6 +63,7 @@ var boostcid = 5721;
 
 
 var EMAILS = [
+    "OVERABILLION",
```

Comment:
And `OVERONEBILLION ` Work for me https://www.reddit.com/r/borderlands3/comments/bxgq5p/borderlands_vip_program_codes/f06vzb8?utm_source=share&utm_medium=web2x